### DISCUSSION TITLE
QR Report Printing

### DISCUSSION THREAD

**Discussion by franciscocontreras93** [opengisch/QField#5564] [Ideas]
Hello! I'm inquiring about the possibility of generating a **QR code** directly within a report that can be printed. This functionality would be highly beneficial.

**Comment by m-kuhn** [comment]
This is an interesting idea, can you add it to our ideas platform https://ideas.qfield.org/app-feature-requests ?

**Reply by franciscocontreras93** [reply]
Done! thanks for you reply!
